bedrock-codec/src/main/java/org/cloudburstmc/protocol/bedrock/packet/TextPacket.java

```diff
@@ -19,6 +19,10 @@ public class TextPacket implements BedrockPacket {
     private List<String> parameters = new ObjectArrayList<>();
     private String xuid;
     private String platformChatId = "";
+    /**
+     * @since v685
+     */
+    private String filteredMessage;
```

Comment:
I think the default value can also be added to the new strings (https://github.com/CloudburstMC/Protocol/pull/226/commits/0ba2458507270f4a73e0d2b7c4d3f3934d9fc1c6#diff-7f9a320683c3f9b0d9a95fe811400e54bfffa9a11fe67f658c81375ee20e6705R177) in the level settings because BDS sends them empty. What do you think about that?